frontend/src/components/ui/question/QuestionBox.js

```diff
@@ -11,24 +11,38 @@ import QuestionCategories from "./QuestionCategories";
 import QuestionTitle from "./QuestionTitle";
 import Question from "./Question";
 
-import styles from './QuestionBox.module.css'
+import styles from "./QuestionBox.module.css";
 
 /* 
   Find out which difficulty user chose,
   obtain a Question of that difficulty from
   Question service and display it
 */
 
-function QuestionBox() {
+function QuestionBox(props) {
```

Comment:
For consistency, can you deconstruct the `props` to `{difficulty, roomId, ...etc}`? It's consider as a good practice in React since it's much cleaner. Thanks!

Reply:
Wow, I did not know about this. Thanks for this information! Btw thanks for changing them under 'Chat' UI!